magento/magento2
# ISSUE-30880 - Add CSP entry to allow google analytics ajax

### Description (*)
Google analytics scripts make ajax calls (to GA). With the existing CSP whitelist in the analytics module, these requests are blocked. This PR adds a CSP entry for connect-src to allow ajax requests to www.google-analytics.com

### Fixed Issues (if relevant)
1. Fixes magento/magento2#30880

### Manual testing scenarios (*)
1. Ensure that Magento_CSP is enabled and functional (content security policy headers appear in store front page responses). 
2. In store configuration, enable Google Analytics and set an account number.
3. Flush caches and visit the store front.
4. In the developer console, confirm that there is no error pertaining to an ajax call to https://www.google-analytics.com/j/collect.
5. In the network tab, confirm that there was a successful ajax call to https://www.google-analytics.com/j/collect.

### Contribution checklist (*)
